tests/ha/hawk_gui.pm

```diff
@@ -18,66 +18,17 @@ use lockapi;
 use hacluster;
 use x11test;
 use x11utils;
-use version_utils 'is_desktop_installed';
-
-sub install_geckodriver {
-    my $python_ver       = shift;
-    my $arch             = check_var('ARCH', 'x86_64') ? 'linux64' : 'linux32';
-    my $geckodriver_from = "https://github.com/mozilla/geckodriver/releases/download";
-    # Using latest stable version of geckodriver depending on which python is being used
-    my $geckodriver_ver = ($python_ver eq 'python3') ? "v0.24.0" : "v0.17.0";
-    my $geckodriver_pkg = "geckodriver-$geckodriver_ver-$arch.tar.gz";
-
-    assert_script_run "wget -P /tmp $geckodriver_from/$geckodriver_ver/$geckodriver_pkg";
-    type_string "cd /tmp\n";
-    assert_script_run "tar -zxvf $geckodriver_pkg";
-    assert_script_run "mv -i geckodriver /usr/local/bin/";
-    type_string "cd\n";
-}
-
-sub install_required_python_pkgs {
-    my $inst_cmd = '';
-    my $pydeps   = 'selenium paramiko';
+use version_utils;
 
-    assert_script_run "zypper in -y python3-pip || zypper in -y python-pip || zypper in -y python-setuptools";
-
-    # Determine how to install python packages
-    if (is_package_installed('python3-pip') or is_package_installed('python-pip')) {
-        $inst_cmd = 'pip install';
-    }
-    elsif (is_package_installed('python-setuptools')) {
-        $inst_cmd = 'easy_install';
-        # When using easy_install, need to force an older version of urllib3 as the newer ones are broken
-        # for easy_install. Will also use a fixed version of paramiko for the same reason
-        $pydeps = 'selenium paramiko==2.1.6';
-        assert_script_run "$inst_cmd -U urllib3==1.23";
-    }
-    die "Couldn't find a way to install python packages" unless ($inst_cmd);
+sub install_docker {
+    my $docker_url = "https://download.docker.com/linux/static/stable/x86_64/docker-19.03.2.tgz";
 
-    # Install paramiko and selenium driver for python and determine which python to use
-    my $output = script_output "$inst_cmd -U $pydeps";
-    my $python = '';
-    if ($output =~ m|.+(python[0-9])[0-9\.]+/site-packages.+|) {
-        $python = $1;    # easy_install way
-    }
-    else {
-        # pip way
-        $output = script_output "pip --version";
-        $output =~ m|.+(python[0-9])[0-9\.]+/site-packages.+|;
-        $python = $1;
-    }
-    die "Couldn't determine which python is installed in the system: [$python]"
-      unless ($python eq 'python2' or $python eq 'python3');
-    return $python;
+    assert_script_run "curl -s $docker_url | tar zxf - --strip-components 1 -C /usr/bin", 120;
+    # Allow the user to run docker. We can't add him to the docker group without restarting X.
+    # The final colon is to avoid a bash syntax error when assert_script_run() appends a semicolon
+    assert_script_run "/usr/bin/dockerd -G users >/dev/null 2>&1 & :";
```

Comment:
It's impossible on SLE4SAP (v12 at least) to activate the Containers Module without converting it into plain SLES.

Reply:
> But this test runs on SLED not in SLES4SAP.

Typo.  The module is not available on SLED.

I just checked and this module is not available even on SLED 15 SP1.

Reply:
We can replace SLED with SLE+Workstation Extension Module and in this case we can add Containers Module.

So what @alvarocarvajald suggested could be done that way I think.

Reply:
> We can replace SLED with SLE+Workstation Extension Module and in this case we can add Containers Module.
> 
> So what @alvarocarvajald suggested could be done that way I think.

In this case we would need SLE + Gnome + WE Module because the tests are no longer run in headless mode.

Reply:
Of course, Gnome was implicit (or "included" in what I call WE :-)).

Reply:
For this I would have post a qcow2 on o.s.d like we have with hawk_test, for both 12 & 15.

I will address this later, now that I have the final fencing test almost done and don't want to break anything.